### ISSUE TITLE
[OutputCache] using CacheManager for WebApi

### ISSUE THREAD

**Issue by Korayem** [MichaCo/CacheManager#100]
This is an amazing nuget but unfortunately it lacks an implementation for OutputCache for WebApi

Other libraries have this out of the box:
https://github.com/filipw/Strathweb.CacheOutput/blob/master/src/WebApi.OutputCache.V2/CacheOutputAttribute.cs
https://github.com/vanthoainguyen/Flatwhite/blob/master/Flatwhite.WebApi/OutputCacheAttribute.cs

Appreciate any pointers to solutions using CacheManager to achieve that

**Comment by MichaCo** [comment]
tl;dr I'm not planning to implement that. If you want to contribute that, feel free to send a PR

Writing just another OutputCache attribute for WebAPI 2 is totally out of scope for CacheManager.
You can probably use one of those implementations and instead of using System.Runtime.Caching directly, create an implementation with CacheManager.

I personally think an OutputCache attribute for WebAPIs is not such a good idea. It makes it easy to use, for sure. But you should have more granular control over what you are actually going to cache in your business logic. It is not really that hard to add caching to most APIs but it depends on the use cases.

In addition to that, in asp.net core, those attribute do not exist either I think.

**Comment by Korayem** [comment]
Thanks @MichaCo for your response and personal opinion about the matter.

Since no [OutputCache] attribute was available and overall, your nuget is superior to others in terms of extensibility, out of the box features and customization flexibility, I already started using it per API inside the code and eventually (and before reading your comment) came to the same conclusion that when it comes to WebAPI there is certain logic that an Attribute might not be flexible enough to accommodate.
